### PR TITLE
Add Experience Platform Connector Extension

### LINKED ISSUE #356: Enable MSE for PWA Studio
* To enable Magento Storefront Events in the PWA parts of the page, we need to install an additional extension.
* [ ] Install https://www.npmjs.com/package/@magento/experience-platform-connector (from https://github.com/magento/pwa-studio/tree/develop/packages/extensions/experience-platform-connector) extension in PWA studio project

### PR DESCRIPTION
**Pull request by herzog31**
This PR adds the Experience Platform Connector extension: https://github.com/magento/pwa-studio/tree/develop/packages/extensions/experience-platform-connector

This extension triggers the following events https://github.com/magento/pwa-studio/tree/develop/packages/extensions/experience-platform-connector/src/handlers (only if they occur in the PWA context).

Normally the extension also includes the AEP setup, but since we do this in Launch/Franklin, I removed that part. See https://github.com/magento/pwa-studio/blob/develop/packages/extensions/experience-platform-connector/src/main.js#L19-L79

Fix #356

Test URLs:
- Before: https://main--maidenform--hlxsites.hlx.page/
- After: https://issue-356--maidenform--hlxsites.hlx.live/

### CODE REVIEW

File: src/components/ExperiencePlatformConnector/ExperiencePlatformConnector.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            }
+            setSdk(mse);
+        });
+    }, [setSdk]);
+
+    useEffect(() => {
+        console.log('sdk', sdk);
+        if (sdk) {
+            const sub = observable.subscribe(async event => {
+                console.log('incoming event', event);
```
Comment (aniham) [inline src/components/ExperiencePlatformConnector/ExperiencePlatformConnector.js:31]: minor: probably remove console statement

**Review by aniham — COMMENTED** [review]
@herzog31 it's hard to follow this PR because of the minified changes and many files that don't seem to have relevant changes for generating events specifically. would it be possible to call out the files that generate cart, account, and checkout events so i can look at those specifically?

**Comment by herzog31** [comment]
@aniham please ignore all files that are in `scripts/pwa-dist`. Because of how Franklin works, we need to commit the PWA build artifacts. So the relevant changes are in 
* package.json (add extension)
* src/components/ExperiencePlatformConnector/ExperiencePlatformConnector.js (create entrypoint for extension, since we don't use `useApp` like PWA)
* src/components/Adapter/adapter.js (our entrypoint)
